packages/cli/src/utils/generate.ts

```diff
@@ -42,9 +44,13 @@ function createTmpFolder() {
 
     mkdirsSync(tmpDir)
   } catch (err) {
-    console.error(err)
+    console.error(`${chalk.red('error')} - ${err}`)
   } finally {
-    console.log(`Temporary folder ${tmpFolderName} created`)
+    console.log(
```

Comment:
This comment is not related to the PR changes itself, but after I saw I thought it was worth pointing it out to see other opinions:

Should we move this logging out of the `finally` block to the end of the `try` block? The `finally` block is also executed [after the `catch` block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#:~:text=These%20statements%20execute%20regardless%20of%20whether%20an%20exception%20was%20thrown%20or%20caught.) so we would have both an error message and a "success" message.
🤔 

This is happening not just in this function but others in this file.

Reply:
Agree with you!  I'll change the code 🎉

Reply:
Done [here](https://github.com/vtex/faststore/pull/1550/commits/8cea4c8d1665cc455c7a3902f21def672f36b63d).